docs/source/release_notes/release-notes.rst

```diff
@@ -23,6 +23,25 @@ Changelog
 .. <<<<------------------------------------------------------------------------------------------------------------->>>>
 
 =======
+v0.7.0
+=======
+
+.. list-table::
+    :widths: 5 40
+    :header-rows: 1
+
+    *   - Type
+        - Change
+
+    *   - |changed|
```

Comment:
This should be breaking and we should create that category if it doesn't already exist.